### ISSUE TITLE
Submit dataset

### ISSUE THREAD

**Issue by jonthegeek** [dslc-io/tidytuesdayR#115]
Add a function, something like `tt_submit_prepare()`, that sets up templating for a TidyTuesday dataset submission, and guides the user through the process. And then a second function, `tt_submit()`, that submits the dataset as a PR.

**Comment by thebioengineer** [comment]
Love the idea!

Just an immediate thought i had was would folks get confused between the similarly named functions? Would it maybe make sense to name the one to submit tt_submission_send() and the one to prepare tt_submission_prepare() or something?

Then the middle term is consistent, and the last one tells you what it will do?

**Comment by jonthegeek** [comment]
Good call! Hopefully I'll find some time to implement these in the next few days. We have a mostly streamlined process now, so I want to make it as easy as possible on the user side!
